website/components/navigation/MobileNavigation.tsx

```diff
@@ -72,7 +72,7 @@ function MobileNavigation({ children }: { children: React.ReactNode }) {
   const [open, setOpen] = useState(false);
 
   const handleBackdropClick = (e: React.SyntheticEvent) => {
```

Comment:
seems this could be typed more precisely as `React.MouseEvent` and wrapped in a `useCallback`

Reply:
👍